.pylintrc

```diff
@@ -0,0 +1,28 @@
+[MESSAGES CONTROL]
```

Comment:
Where's the change when these linters are applied to all the files in the project. I think I will delay merging this just to have less conflicts with other open PRs.

Reply:
I had just made the files and ran them only on few files and not all. Should I run them on all the files?

Reply:
@BassCoder2808 Did you run the tests for this after linting?